ses-lambda-sam/src/process-email.py

```diff
@@ -0,0 +1,22 @@
+import json
+
+# Create an AWS Lambda function that processes emails received by Amazon SES
+def lambda_handler(event, context):
+    # Get the email message from the event data
+    message = event['Records'][0]['ses']['mail']
```

Comment:
Can their be multiple records?

Reply:
There will be one record as each email triggers the Lambda function.